source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -2824,7 +2831,7 @@ export class ToggleInput extends Input {
         Utils.appendChild(element, this._checkboxInputElement);
 
         if (this.title || this.isDesignMode()) {
-            let label = new Label();
+            let label = new TextBlock();
```

Comment:
Is this right? Didnt we say label for v1 Inputs would just be a string type? @paulcam206?

Reply:
Yes, it is right. I'm using a TextBlock internally to render because it holds all the logic, but it remains that the `label` property is just a string.